pkg/devfile/parse_test.go

```diff
@@ -46,7 +46,7 @@ func TestParseDevfileAndValidate(t *testing.T) {
 
 	devfileStruct := schema.DevWorkspaceTemplate{
 		TypeMeta: metav1.TypeMeta{
-			APIVersion: "2.1.0",
+			APIVersion: "2.3.0",
```

Comment:
Instead of moving this to `2.3.0` what do you think about having a struct for each version to test against, it looks to me like we are moving to only testing `2.3.0`?

Reply:
we don't need to update all the tests to be `2.3.0`,  library is, and should be, backward compatible, so for any existing features we test with, leave it as it's previous version.

Reply:
only bump-up devfile version in test if the unit test covers new features introduced in the latest version
@michael-valdron

Reply:
FYI, just created this issue: https://github.com/devfile/api/issues/1603

Reply:
> only bump-up devfile version in test if the unit test covers new features introduced in the latest version @michael-valdron

@yangcao77 @Jdubrick Reverted with https://github.com/devfile/library/pull/219/commits/267aa8bccf389f2625c41034853e4822d801f921, though I left in the changes to the `unsupportedSchemaError` test case since the result will change with the addition of `2.3.0`.